src/Tribe/Tickets.php

```diff
@@ -937,11 +946,11 @@ public function supports_global_stock() {
 		}
 
 		/**
-		 * Returns instance of the child class (singleton).
+		 * Returns class instance. Child classes must overload this.
 		 *
 		 * @static
 		 *
-		 * @return mixed
+		 * @return static
```

Comment:
:no_entry_sign: Function return type is not void, but function has no return statement

<sub>hash: 0b456e2456a3279850f526d787e6e6bb</sub>